src/pydot/dot_parser.py

```diff
@@ -372,12 +372,9 @@ def graph_definition():
         rbrace = Literal("}")
         lbrack = Literal("[")
         rbrack = Literal("]")
-        lparen = Literal("(")
-        rparen = Literal(")")
         equals = Literal("=")
         comma = Literal(",")
         semi = Literal(";")
-        at = Literal("@")
         minus = Literal("-")
```

Comment:
This is tangential, but all of these `Literal()` aliases actually really annoy me. Unlike the `CaselessKeyword()`s that precede them, which _are_ needed for case-insensitive parsing, explicit `Literal()`s are **totally** unnecessary.

Just using a bare string value in a pyparsing expression **implies** `Literal()`. So, `"="` is equivalent to `Literal("=")` in every way except that it isn't a `ParserElement`, so you can't call methods on it like `"=".suppress()`.

But that's not necessary anyway, because `Suppress()` exists, and like every other pyparsing grammar class is a subclass of `ParserElement`. So IMHO, the grammar is more readable when a rule like this,

```python
graph_stmt = Group(
    lbrace.suppress()
    + Optional(stmt_list)
    + rbrace.suppress()
    + Optional(semi.suppress())
).setName("graph_stmt")
```
is instead written as,

```python
graph_stmt = Group(
    Suppress("{") + Optional(stmt_list) + Suppress("}") + Optional(";").suppress()
).setName("graph_stmt")
```
or even (not sure if this would work):

```python
    Suppress("{") + Optional(stmt_list) + Suppress("}" + Optional(";"))
```

Reply:
Should've known that'd be too wide for GitHub's narrow comment rendering.

Reply:
Noted for the future.